### DISCUSSION TITLE
compatiablity with postgres 13 ?

### DISCUSSION THREAD

**Discussion by FarisZR** [simple-login/app#460] [Self-hosting question] [answered]
hi 
i am using a modified postgressql 13 alpine image with multi user support

i noticed simplelogin uses pgsql 12 
is there an issue with version 13 ? or i can go ahead  and use it without any problems ?

**Comment by nguyenkims** [comment, marked as answer]
SimpleLogin doesn't use any particular feature of Postgres 12 so I think it'd be safe to upgrade to Postgres 13. We never use Postgres 13 in production though.

**Comment by FarisZR** [comment]
can confirm its working fine now, no problems so far
